pkg/network/handlers/work_package_sharing.go

```diff
@@ -9,19 +9,36 @@ import (
 
 	"github.com/quic-go/quic-go"
 
+	"github.com/eigerco/strawberry/internal/authorization"
 	"github.com/eigerco/strawberry/internal/common"
+	"github.com/eigerco/strawberry/internal/crypto"
+	"github.com/eigerco/strawberry/internal/refine"
+	"github.com/eigerco/strawberry/internal/service"
 	"github.com/eigerco/strawberry/internal/work"
 	"github.com/eigerco/strawberry/pkg/network/peer"
 	"github.com/eigerco/strawberry/pkg/network/protocol"
 	"github.com/eigerco/strawberry/pkg/serialization/codec/jam"
 )
 
 type WorkPackageSharer struct {
-	guarantors []*peer.Peer
+	guarantors   []*peer.Peer
+	Auth         authorization.AuthPVMInvoker
```

Comment:
Same here, why do we export these fields?

Reply:
there is no need, i have made them private